packages/pds/src/api/com/atproto/repo/putRecord.ts

```diff
@@ -14,7 +14,11 @@ import {
 } from '../../../../repo'
 import { ConcurrentWriteError } from '../../../../services/repo'
 
-const ALLOWED_PUTS = [ids.AppBskyActorProfile, ids.AppBskyGraphList]
+const ALLOWED_PUTS = [
+  ids.AppBskyActorProfile,
+  ids.AppBskyGraphList,
+  ids.AppBskyFeedGenerator,
```

Comment:
Updates to this record time should get indexed just fine, might be worth a double-check though.

Reply:
yup yup i added a quick tests